documents/Libraries/sxpbrlib/sx-glsl/sx_normalmap.glsl

```diff
@@ -1,6 +1,16 @@
-void sx_normalmap(vec3 value, vec3 N, vec3 T, out vec3 result)
+void sx_normalmap(vec3 value, int interpretation, vec3 N, vec3 T, out vec3 result)
```

Comment:
As mentioned offline, we should probably change "interpretation" to a string called "space" for consistency.